tests/subsys/logging/log_msg/src/log_msg_test.c

```diff
@@ -20,11 +20,16 @@ extern struct k_mem_slab log_msg_pool;
 static const char my_string[] = "test_string";
 void test_log_std_msg(void)
```

Comment:
maybe you could rework this test a bit and create a loop that tests up to six arguments and determines expected number of chunks allocated like `(arg > LOG_MSG_NARGS_SINGLE_CHUNK) ? 2 : 1`. That would look better and would not need ifdef.